portfolio/src/main/java/com/google/sps/servlets/CommentsServlet.java

```diff
@@ -13,6 +13,12 @@
 // limitations under the License.
```

Comment:
If I hard refresh your page (cmd+shift+r) it takes several seconds to load now. Not important for this CL, but it might be worth thinking about how to make that faster.

Reply:
Will work on this!